pages/guestbook.js

```diff
@@ -23,15 +23,25 @@ export default function GuestbookPage({ initialEntries }) {
 }
 
 export async function getStaticProps() {
-  // this needs to be able to be serialized
-  // const [rows] = await db.query(`
-  //   SELECT id, body, created_by FROM guestbook
-  //   ORDER BY id;
-  // `);
+  const [rows] = await db.query(`
+    SELECT * FROM guestbook
+    ORDER BY id;
+  `);
+
+  const entries = rows.map(
+    ({ id, email, body, created_by, created_at, updated_at }) => ({
+      id: id,
```

Comment:
Redundant `id: id,` here, can be simplified to just `id,`. See [MDN page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#new_notations_in_ecmascript_2015) on ES6 shorthand property names.

Reply:
Good call! I wonder if there's an even easier way to "serialize" this data. Will dig more 😄

Reply:
@leerob 

try this

```js
const entries = Object.values(JSON.parse(JSON.stringify(rows)));
```

if this is the node MySql driver, then this should convert the raw packets to a JSON string, then parse it as JSON objects and get their values.

after that you can loop the entries and display

Reply:
Thank you! Seems to work.